Smaller bestmove

GediminasMasaitis opened the pull request:
-5 bytes

2 out of the 5 are from replacing `Move` to `auto`

GediminasMasaitis committed:
Smaller bestmove

XInTheDark commented on README.md:
````diff
@@ -28,7 +28,7 @@ bash build-mini.sh
 ## 4ku-mini Size
 ```
-3,901 bytes
+3,986 bytes
````
There should be a typo here, 3901-5 is 3896 :)

GediminasMasaitis replied:
Thanks, fixed :)

cj5716 committed:
-5 bytes in zobrist hashing (#212)

This patch was inspired by crippa1337/Pytteliten#33

-5 bytes
Bench 5224614

cj5716 committed:
Bench 5224614 (#214)

Replace 1 - improving with !improving

-2 bytes
Bench: 5224614
No functional change

GediminasMasaitis committed:
Merge branch 'master' into smaller_bestmove

GediminasMasaitis committed:
Update size

GediminasMasaitis committed:
Merge branch 'master' into smaller_bestmove

GediminasMasaitis committed:
Merge

GediminasMasaitis committed:
Merge branch 'master' into smaller_bestmove

GediminasMasaitis committed:
Merge
